includes/class-amp-travel-blocks.php

```diff
@@ -12,6 +12,13 @@
  */
 class AMP_Travel_Blocks {
 
+	/**
+	 * Required count for featured locations block.
+	 *
+	 * @var int
+	 */
+	public static $featured_locations_count = 6;
```

Comment:
This class is instantiated, so would there be a need to make this static? If not, could you please remove `static`, and change this to something like:
```php
const FEATURED_LOCATIONS_COUNT = 6;
```

Then, references to it could change from `self::$featured_locations_count` to `self::FEATURED_LOCATIONS_COUNT`

This is outside the diff of this PR, but could you please do the same thing to `$popular_posts_count` [below](https://github.com/xwp/travel/pull/34/files#diff-d85bb5c137cc4b60d13bd558d64a58d9R27)?